src/components/mdCheckbox/mdCheckbox.scss

```diff
@@ -19,6 +19,7 @@ $checkbox-touch-size: 48px;
 
     &:focus {
       outline: none;
+      box-shadow: 0 0 0 2px rgba(#000, .14);
```

Comment:
so, i previously did that (at least made it look like that). 
Thing is: toggling the checkbox will not remove the focus even when clicking, so the active ripple will look rather strange.